src/FakeCLI.java

```diff
@@ -0,0 +1,2 @@
+public class FakeCLI {
+}
```

Comment:
Only add/commit things that you need and relate to the commit. In this case `FakeCLI`, `Main` and `UserInterface` are not used anywhere thus have no purpose in this commit.